lib/src/ovis_util/util.c

```diff
@@ -990,21 +990,53 @@ const char* ovis_errno_abbvr(int e)
 	return "UNKNOWN_ERRNO";
 }
 
+/** should be bigger than any expected strerror string.
+ * In 2020 ~50 was the upper limit.
+ */
+#define OVIS_SYS_ERRLEN 80
+
+/** should be bigger than any expected errno value.
+ * In 2020 ~140 was the upper limit.
+ */
+#define OVIS_SYS_NERR 256
+
+static char ovis_sys_errlist[OVIS_SYS_NERR][OVIS_SYS_ERRLEN];
+static int ovis_sys_errlist_initialized;
+static int ovis_sys_nerr;
+/* glibc has dropped support for sys_errlist.
+ * we copy it via strerror on first call.
```

Comment:
This could use a better explanation. We can't copy something that isn't supported. Instead describe what it actually does, which is build an array of error strings by calling strerror_r() for all possible error values.

And "on first call" doesn't apply to this function, it applies to ovis_strerror().

Reply:
this function is externally uncallable in the previous case. I just now made it callable as ovis_sys_errlist_init, but its use is optional. At least the name should now be clear when calling it from main()s.